Code/app/src/main/java/com/cmpt362/fitcheck/firebase/Firebase.kt

```diff
@@ -1,12 +1,12 @@
 package com.cmpt362.fitcheck.firebase
 
 import android.net.Uri
+import androidx.lifecycle.MutableLiveData
 import com.google.android.gms.tasks.Task
 import com.google.firebase.auth.AuthResult
 import com.google.firebase.auth.FirebaseAuth
 import com.google.firebase.auth.ktx.auth
-import com.google.firebase.database.DatabaseReference
-import com.google.firebase.database.FirebaseDatabase
+import com.google.firebase.database.*
 import com.google.firebase.database.ktx.database
```

Comment:
Since you are importing using `com.google.firebase.database.*`, I think we don't need line 10

Reply:
we still need line 10 cause when I get rid of it we get some import errors,